frontend/src/metabase-types/types/Dataset.ts

```diff
@@ -26,6 +26,7 @@ export type Column = {
   expression_name?: any;
   settings?: ColumnSettings;
   field_ref?: Field;
+  remapped_to_column?: Column;
```

Comment:
This is a legacy type which is deprecated. The new one is here `frontend/src/metabase-types/api/dataset.ts`